src/Microsoft.AspNet.Server.Kestrel/KestrelServerInformation.cs

```diff
@@ -17,6 +17,10 @@ public class KestrelServerInformation : IKestrelServerInformation, IServerAddres
 
         public bool NoDelay { get; set;  }
 
+        public int MaxHeaderBytes { get; set; } = 16384; // 16kB
```

Comment:
Something is out of sync with this PR. This property is read from config but the server never checks it.